--- include/backend/input.h.orig	2020-07-08 20:50:58.201435840 +0200
+++ include/backend/input.h	2020-07-08 20:51:58.847433821 +0200
@@ -43,6 +43,6 @@
 
 	int hotspot_x;
 	int hotspot_y;
-} pointer;
+};
 
 void input_release(struct input *S);
--- src/backend/input.c.orig	2020-07-08 20:52:08.169433511 +0200
+++ src/backend/input.c	2020-07-08 20:52:25.543432933 +0200
@@ -15,6 +15,8 @@
 #include <linux/input.h>
 #include <xkbcommon/xkbcommon.h>
 
+struct pointer_ pointer;
+
 struct input {
 	int key_fd;
 	int pointer_fds[16];
--- src/swvkc.c.orig	2020-07-08 20:52:43.424432338 +0200
+++ src/swvkc.c	2020-07-08 20:52:53.423432005 +0200
@@ -28,6 +28,8 @@
 #include <string.h>
 #include <strings.h>
 
+extern struct pointer_ pointer;
+
 struct server {
 	struct wl_display *display;
 
